### DISCUSSION TITLE
Why am I getting this BS error about `Button`

### DISCUSSION THREAD

**Discussion by PtradeLLC** [shadcn-ui/ui#3023] [General]
So, I followed the installation guide for NextJs with the spirit of using `npm` to seamlessly install shadcn in my app. But, while trying to run the app I got an error:
```
Failed to compile
./components/ModeToggle.js:5:0
Module not found: Can't resolve '@/components/ui/button'
  3 | import { useTheme } from "next-themes"
  4 |
> 5 | import { Button } from "@/components/ui/button"
  6 | import {
  7 |     DropdownMenu,
  8 |     DropdownMenuContent,

https://nextjs.org/docs/messages/module-not-found

Import trace for requested module:
./components/Navbar.js
./components/Layout.js
./pages/_app.js
```
Um... Why couldn't shadcn just include all these components along with the package rather than having folks go through the troubles of manually installing one componenyy at a time. Stop being lazy - Geez. Having Dark mode isn't worth the trouble thank you.

**Comment by dertuman** [comment]
I'm having the same issue, so basically it only fails for me on the CI. It doesn't fail on a digital ocean droplet with ubuntu on it or locally, and I'm stuck!!!

**Comment by Mssskrishna** [comment]
How did you fix this problem?

**Comment by josephbima** [comment]
Having the same problem

**Comment by josephbima** [comment]
Okay I fixed this by adding this

`  "compilerOptions": { "baseUrl": ".", "paths": { "@/*": ["./src/*"] } }
`

in your jsconfig.json

This was discussed in the issue https://github.com/shadcn-ui/ui/issues/1101
